### ISSUE TITLE
Version 0.12.1 does not work with production builds

### ISSUE THREAD

**Issue by marekrozmus** [typestack/class-validator#588]
When running `npm prune --production` following warning is displayed: `npm WARN class-validator@0.12.1 requires a peer of tslib@>=1.9.0 but none is installed. You must install peer dependencies yourself.`

When running built app following error appears:
```
Error: Cannot find module 'tslib'
server_1    | Require stack:
server_1    | - /usr/src/app/node_modules/class-validator/bundles/index.umd.js
```

With version 0.11.1 same code/setup works ok.

**Comment by vlapo** [comment]
Sorry I tested `peerDependencies` and forgot move tslib into `dependencies` before release.

**Comment by vlapo** [comment]
Fixed in `0.12.2` 🚀
